pkg/virt-operator/resource/apply/apps.go

```diff
@@ -276,7 +276,7 @@ func (r *Reconciler) syncDaemonSet(daemonSet *appsv1.DaemonSet) (bool, error) {
 
 	injectOperatorMetadata(kv, &daemonSet.ObjectMeta, imageTag, imageRegistry, id, true)
 	injectOperatorMetadata(kv, &daemonSet.Spec.Template.ObjectMeta, imageTag, imageRegistry, id, false)
-	InjectPlacementMetadata(kv.Spec.Workloads, &daemonSet.Spec.Template.Spec)
+	InjectPlacementMetadata(kv.Spec.Workloads, &daemonSet.Spec.Template.Spec, AnyNode)
```

Comment:
Similar to the comment about Deployment, here its seems the same for DaemonSets, does it make sense to limit this change to virt-handler only, that should be deployed on worker nodes only?

Reply:
This is being limited only to daemon sets, and virt-handler is our only daemon set. It is the only exception.